cloud/amazon/ec2_elb_lb.py

```diff
@@ -360,6 +360,29 @@
 except ImportError:
     HAS_BOTO = False
 
+import time
+import random
+
+def _throttleable_operation(max_retries):
```

Comment:
wow. this seems like an enormously valuable helper function. seems like it would be more useful farther down in the library where many modules could leverage it. i see [this](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/api.py#L82) and wonder if we could leverage it, or if we should just have a boto specific implementation in the [ec2 module utils](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/ec2.py).

either way, i wouldn't block this PR going through (as it seems useful) but would suggest something like this deeper in the library.